### ISSUE TITLE
wont work in production

### ISSUE THREAD

**Issue by pYr0x** [stealjs/cache-bust#12]
as worte in http://forums.donejs.com/t/problems-getting-stealjs-cache-bust-to-work/514/4
steal-cache-bust can (today) not work in production.

in production steal-cache-bust is bundled into the index.js file and steal therefore cant be load the index.js file with the cache-bust extension because it dont know that cache-busting is enabled.

for any future files, steal will load these files with a version number

@matthewp 
my thoughts:
either we integrate steal-cache-buster into core and enable it by setting the steal config
or
we add (in steal 1.0) the fetch hook into the steal.production.js that we copied into the dist folder and remove cache-buster from being bundled

**Comment by matthewp** [comment]
Yeah, this is a known limitation. I wonder if it would be better if steal-cache-bust didn't get bundled at all, and to use it you would add a second script tag like:

```html
<script src="path/to/steal/steal.production.js"></script>
<script src="path/to/steal-cache-bust/cache-bust.min.js"></script>
```

I think that makes more sense personally.

**Comment by pYr0x** [comment]
in steal-tools 1.0 we create a pre configured steal production version.
why cant we add into this pre configured steal, the cache bust?

**Comment by matthewp** [comment]
We could, yeah.  We'd need some type of API for that though. Don't want to special-case steal-conditional.

**Comment by pYr0x** [comment]
any idea is welcome ;)
btw: we can also can handle the the NPM stuff, that we currently add into a bundle, this way.

**Comment by tai101568** [comment]
Hi matthewp,
I don't see cache-bust.min.js file in folder steal-cache-bust.

**Comment by matthewp** [comment]
That's right, it doesn't exist yet @tai101568, it's just one possible solution to this problem.

**Comment by tai101568** [comment]
how to get it, matthewp? :) copy code in cache-bust.js to new js file?

**Comment by matthewp** [comment]
I don't think I follow your question? We haven't really arrived at a great solution to this problem yet.

**Comment by pYr0x** [comment]
@tai101568 here a workaround until we finialize the issue
i created a gist https://gist.github.com/pYr0x/e9802368d696adcbaa8bfe478fccf0dd
you can use it like matthew described here https://github.com/stealjs/cache-bust/issues/12#issuecomment-269327725

**Comment by pYr0x** [comment]
@matthewp you want to talk about this next week?

**Comment by matthewp** [comment]
Yep, sure no problem!
